### DISCUSSION TITLE
Graphql support for Meta data (aggregation)

### DISCUSSION THREAD

**Discussion by dukesx** [directus/directus#4636] [Archive]
I am looking at the docs, it seems that there is no example for GraphQL supporting aggregation ? or is it that the docs are outdated ? 🤔 By Aggregation i mean "total count" / "filter count" that is supported in the Rest Api. 

Someone put a light on this please.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/4247
